### PR TITLE
Initialize bbox with first element

### PR DESCRIPTION
**Pull request by nnmm**
The bounding box is wrong for point clouds that don't contain (0, 0, 0). This solution may result in a small performance hit. Should I add a TODO to add a next() method to the stream? Then we can get the first point to initialize the bbox from that without adding a check inside the loop.

### CODE REVIEW

**Comment by nnmm** [comment]
ptal @catevita

File: src/generation.rs [hunk 1]
```diff
@@ -260,6 +260,10 @@ fn find_bounding_box(input: &InputFile) -> Aabb3<f32> {
     }
 
     stream.for_each(|p: &Point| {
+        if num_points == 0 {
+            let p3 = Point3::from_vec(p.position);
+            bounding_box = Aabb3::new(p3, p3);
```
Comment (catevita) [inline src/generation.rs]: It works, but how about initializing the bounding box with the first two points and then entering the loop? so we can spare the integer comparison during the loop and remove the erroneous initialization in line 256.

Comment (catevita) [reply src/generation.rs]: i noticed you mentioned the same thing on the description!

Comment (SirVer) [reply src/generation.rs]: This seems fine to me. Bounding box calculation is not the time critical piece, and the branch will run very hot in branch predcition quickly, so performance will not suffer either. Let's have it that way.

**Comment by SirVer** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user SirVer. Merge queue now has a length of 1.

**Pull request merged**